### ISSUE TITLE
Support dask dataframe in train_test_split

### ISSUE THREAD

**Issue by TomAugspurger** [dask/dask-ml#351]
Easy way: call `to_dask_array(lengths=True)`. This will take some computation.

The harder (maybe not too hard) way to do this would be to directly support dask dataframes.

**Comment by mrocklin** [comment]
Perhaps http://dask.pydata.org/en/latest/dataframe-api.html#dask.dataframe.DataFrame.random_split can be useful here?

**Comment by TomAugspurger** [comment]
Yeah, I think so. Will just take a bit of work to ensure that we split multiple dataframes the same.

**Comment by mrocklin** [comment]
You can provide a seed to the method and it will split them the same way
(assuming that the chunk structure is the same)
